spec/site/alarm_spec.rb

```diff
@@ -67,4 +67,43 @@ def check_scripts
   #    system(SCRIPT_PATHS['deactivate_alarm'])
   #  end
   #end
+
+  it 'Disruption test', :script, sxl: '>=1.0.7' do |example|
```

Comment:
test name should be more descriptive, maybe something like "buffers alarms during disconnects"

Reply:
Sure, sounds good to me! I'll change it.